integration_tests/integration/allocation-complete.spec.ts

```diff
@@ -9,7 +9,7 @@ context('Allocate Complete', () => {
     cy.task('stubSetup')
     cy.task('stubGetStaffByCode')
     cy.task('stubSearchStaff')
-    cy.task('stubGetCurrentlyManagedCaseOverview', '1')
+    cy.task('stubGetCurrentlyManagedCaseOverview')
```

Comment:
for some reason the stub parameter was still set to `convictionId = '123456789'` and all references to this stub set the parameter at 1 so changed it